### ISSUE TITLE
Multi-page note disappears from existence

### ISSUE THREAD

**Issue by 0oWow** [streetwriters/notesnook#38]
Last night I typed a multi-page note that is to be a talk I'm giving soon, but now when I check Notesnook this morning, I find the note to be nowhere, not even on the browser I initially typed the note (browser was closed for the night after typing the note).  Not even in the trash.  I did not check the mobile apps after typing the note to see if they were synced before going to bed, since I had no errors, I assumed all was well.

I tested typing fake notes this morning into the same web browser and they work, but that one critical note is missing.  Fortunately I backed up the note with google docs and a downloaded copy to my computer before shutting down yesterday.

I was given no errors to indicate sync failure.  If I can't trust the app to keep the note I've typed, then the app is useless.  In the interest of you being a new app, I would like to help you out where I can.

When I typed the note, I used Firefox version 92.0, beta channel, x64 bit, at app.notesnook.com.  I do have Ublock Origin, but I had your domains whitelisted, as they still are.  Also, the blocking I have in Firefox is only set to third-party cookies.  That doesn't seem to be an issue, as I can type notes this morning and it synced fine.  

I have an Android app and an iPhone app, and between the three devices (web, Android, and iPhone), the critical note is nowhere.

Not sure what you're able to see, but the Notesnook email account is the same as this Github account.

**Comment by thecodrr** [comment]
Hey, this is a weird issue but it looks similar to another issue a user reported a few days back. Basically, what perhaps happened is that due to automatic syncing running in the background, the note you were editing went into a blindspot i.e. the syncing system couldn't detect if it was new or old and skipped it.

Now the weird part is that you were editing it constantly (were you?) so it shouldn't have happened since the blindspot is destroyed if a note is edited again.

This blindspot issue has been fixed but not released yet.

**Comment by 0oWow** [comment]
Yes, I had been editing that note for over an hour, and made several
changes to the document.

I am a system admin and PC tech, am I am used to paying attention to the
little details in the software transactions, but I did not see any unusual
activity, other than it never indicates that it is syncing the note.  It
would be really nice to have a "note synced" indicator, especially one that
was verified to be accurate.

**Comment by 0oWow** [comment]
Apologies, I do see the "synced time" message at the bottom of the notes now.  However, I don't if it showed up for that note or not.  If it did, then it wasn't accurate.

I just copy/pasted the entire note back into the app using Firefox and it seems to have synced successfully this time.

**Comment by 0oWow** [comment]
OK, so I just made some editing changes to the new note, only changing the color of two sentences.  When I looked at the "sync" message at bottom left, it just says "synced 4 minutes ago", but at the bottom right, it says "saved".  However, I closed the tab and performed syncing on my Android app, and as expected, the color changes were not carried over.

**Comment by 0oWow** [comment]
Tested once again, making color changes to the new note.  Did I miss documentation somewhere that says we have to manually sync?  It refused to automatically sync, only saying "synced 9 minutes ago", but after manually clicking "sync" that synced correctly.

**Comment by 0oWow** [comment]
OK, I tested in Vivaldi and I see that it syncs correctly, when I click out of the document, but Firefox never syncs unless you do it manually.  I believe that is the problem.  I just created a new note in Firefox and it has not yet sync'd, even several minutes later, which tells me that this note is not on the server and will be lost if I close the page.

**Comment by 0oWow** [comment]
![Screenshot_20210903_161647](https://user-images.githubusercontent.com/65820375/132066193-b05d130d-d2c6-45fc-839f-dc3e4970bf33.png)

Notice the "test" timestamp of 31 minutes ago, versus the last sync time, 37 minutes ago.

**Comment by 0oWow** [comment]
*sigh* I think I found the problem in Firefox.  When I disabled "enhanced tracking protection" for the page, now Sync is behaving correctly (not immediately as I feel it should be though).  Interestingly, "enhanced tracking protection" was only set to block third-party cookies.

**However**, now I see that the app has replaced my new note, the note that I mentioned in the OP, but that I reentered today.  It replaced the note with your welcome note, all except for my note's title.  

That just doesn't make any sense at all.  I did no such copy/pasting.
